php/deepgram_transcribe_audio/index.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+require 'vendor/autoload.php';
+
+use GuzzleHttp\Client;
+
+return function($req, $res) {
+    try {
+        $payload = \json_decode($req['payload'], true);
+        if (!isset($payload['fileUrl']) || empty($fileUrl = \trim($payload['fileUrl']))) {
+            throw new \Exception('Please provide valid file url.');
```

Comment:
Please make sure not to throw exceptions. and return an error response like:

```json
{"success":false,"message":"Please provide a valid file URL."}
```

instead.